rmw_fastrtps_cpp/include/rmw_fastrtps_cpp/TypeSupport.hpp

```diff
@@ -56,11 +58,11 @@ struct StringHelper<rosidl_typesupport_introspection_c__MessageMembers>
   {
     auto c_string = static_cast<rosidl_generator_c__String *>(data);
     if (!c_string) {
-      fprintf(stderr, "Failed to cast data as rosidl_generator_c__String\n");
+      RCUTILS_LOG_ERROR("Failed to cast data as rosidl_generator_c__String")
       return "";
     }
     if (!c_string->data) {
-      fprintf(stderr, "rosidl_generator_c_String had invalid data\n");
+      RCUTILS_LOG_ERROR("rosidl_generator_c_String had invalid data")
```

Comment:
oh actually I put named only in places where it was already emulating a named logger. But yeah we can use named everywhere :+1: